src/utils/moonpay.js

```diff
@@ -0,0 +1,39 @@
+import sendJson from 'fetch-send-json'
+import { ACCOUNT_HELPER_URL } from '../utils/wallet'
+
+const MOONPAY_API_URL = process.env.MOONPAY_API_URL || 'https://api.moonpay.com'
```

Comment:
Looks like SetupImplicit still contains following code instead of using this module